### DISCUSSION TITLE
How can I disable basic auth?

### DISCUSSION THREAD

**Discussion by adriaurora** [crazy-max/docker-rtorrent-rutorrent#389] [Q&A]
Hi, as I'm using Authelia I want to disable the basic auth login but I'm not being able to make it. I've tried to delete rutorrent.htpasswd or by deleting its content bbut nnothing works.

Thanks in advance.

**Comment by unsubscrieb** [comment]
I had the exact same scenario, comment out the `.htpasswd` mount in `docker-compose.yml` and I found it no longer prompts for basic auth
